### PR TITLE
 Refactor Python scripts according to PEP 8

### PR DESCRIPTION
**Pull request by alanyee**
-Implement best practices for python code
-Follow style guidelines from pep 8
-Other cleanup such as removing unused imports and variables

### CODE REVIEW

**Comment by alanyee** [comment]
@evonide I went around trying to revert the commit while trying to sidestep around the privacy email setting. I realized it would be faster to just nuke the repo and figure it out for next time.

File: crawl_patches.py [hunk 1]
```diff
@@ -14,23 +14,27 @@
...
+from colorama import Fore, Style
 from pygments import highlight
 from pygments.formatters import TerminalFormatter
 from pygments.lexers import SqlLexer
 
+from lib.vcs_management import get_vcs_handler
+from data.models import Nvd, Reference, Vulnerability, VulnerabilityGitCommits
+from data.database import DEFAULT_DATABASE, init_app as init_db
+
+sys.path.append('third_party/')
```
Comment (bluec0re) [inline crawl_patches.py]: the sys.path.append call needs to happen before the other imports

File: main.py [hunk 2]
```diff
@@ -14,30 +14,30 @@
...
+from app.vuln import bp as vuln_bp
+from app.vcs_proxy import bp as vcs_proxy_bp
+from app.vulnerability import VulncodeDB
+import cfg
 from data.database import DEFAULT_DATABASE, init_app as init_db
+from lib.utils import manually_read_app_config
+
+if not 'MYSQL_CONNECTION_NAME' in os.environ:
+  print('[~] Executed outside AppEngine context. Manually loading config.')
+  manually_read_app_config()
```
Comment (bluec0re) [inline main.py]: This block needs to be executed before any other app.*, cfg or lib.* imports.

**Comment by alanyee** [comment]
@bluec0re alright, I think that should resolve problem.

**Comment by evonide** [comment]
Sorry for the delay Alan and thanks again for your contribution :)!

**Pull request merged**